app/reflexes/white_label_reflex.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class WhiteLabelReflex < ApplicationReflex
+  include EnterpriseConcern
+  delegate :view_context, to: :controller
+
+  def remove_logo
+    @enterprise.update!(white_label_logo: nil)
+
+    f = ActionView::Helpers::FormBuilder.new(:enterprise, @enterprise, view_context, {})
+
+    html = with_locale {
+      render(partial: "admin/enterprises/form/white_label",
+             locals: { f: f, enterprise: @enterprise })
+    }
+    morph "#white_label_panel", html
```

Comment:
I didn't pick up on this before, but just noticed that this part is simply re-rendering the white_label form area. Excuse me while I muse...

This seems like it would be a common pattern for reflexes, ie:
1. make a change
2. re-render the form
3. display a response

For re-rendering the form, we could probably add some structure around it. To begin with, I'd made it a separate method (it could be re-used by other reflexes in the future).

But I can also imagine in the future having more 'rails magic' that works by default, just like a rails controller knows how to render a view.

Right now, I don't suggest changing, but I think we should keep looking for ways to add helpful structure to the codebase..

Reply:
Ah, so you mean that we could have a helper like this:

```
    morph_element "white_label_panel", locals: { f: f, enterprise: @enterprise }
```

And it takes the first parameter as `id` and partial name. The rest is passed on to `render`? There may be other parameters we would need to consider but the idea is good. Hm, or should we just use component reflexes then?

Reply:
>  Hm, or should we just use component reflexes then?

I forgot about that! I need to get into this some more.